client/src/components/App/App.js

```diff
@@ -1,28 +1,29 @@
-import React from "react";
-import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
-import "antd/dist/antd.css";
-
-import EventPage from "../container/user/EventPage";
+import React from 'react';
+import { BrowserRouter as Router, Route, Switch ,Link} from 'react-router-dom';
 import Landing from "../container/user/LandingPage";
-import Register from "../container/user/Registeration";
+import Dashboard from "../container/admin/controlPage/MainPage"
+import Register from '../container/user/Registeration'
+import EventPage from "../container/user/EventPage";
+
+import "antd/dist/antd.css";
 import "./App.css";
 
 const App = () => {
   return (
     <div className="App">
       <Router>
+        <Link to="/admin/dashboard">dashbard</Link>
         <Switch>
           <Route
             exact
             path="/events/:eventProg/:eventCode"
             component={EventPage}
           />
           <Route exact path="/" component={Landing} />
-          <Route
-            exact
-            path="/register/:eventProg/:eventCode/:mobile"
-            component={Register}
-          />
+          <Route exact path="/admin/dashboard" component={Dashboard} />
+          <Route exact path="/register/:eventProg/:eventCode/:mobile" 
+          component={Register}>
+      </Route>
```

Comment:
make self closing here
```suggestion
          <Route exact path="/register/:eventProg/:eventCode/:mobile" 
          component={Register} />
```